apps/spruce/.env-cmdrc.json

```diff
@@ -0,0 +1,72 @@
+{
+  "local": {
+    "NODE_ENV": "production",
```

Comment:
Could you file a ticket about the `NODE_ENV` bug if there isn't one already? Is very weird

Reply:
I wonder now if there's a chance `env-cmd` was clobbering Vite's `NODE_ENV` and perhaps `--no-override` would fix that. I'll leave a followup comment in the ticket :)

Reply:
created: [DEVPROD-11638](https://jira.mongodb.org/browse/DEVPROD-11638)